.github/workflows/docker_login.yml

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: Docker login
+
+on:
+  push:
+    branches:
+      - master
+      - tanvi/new_branch
+
+jobs:
+  build:
+    name: docker login
+    runs-on: ubuntu-latest
```

Comment:
Why are you creating a new workflow? You probably need to change the existing workflow to do a login before doing docker pulls.